data/migrations/versions/b82361fba1cd_create_table_for_org_to_sku_relations.py

```diff
@@ -0,0 +1,38 @@
+"""Create table for org to sku relations
+
+Revision ID: b82361fba1cd
+Revises: 46980ea2dde5
+Create Date: 2023-06-07 14:22:09.384808
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = "b82361fba1cd"
+down_revision = "a0fc06d604a4"
+
+from typing import Text
+
+import sqlalchemy as sa
+
+
+def upgrade(op, tables, tester):
+    op.create_table(
+        "organizationrhskus",
+        sa.Column("id", sa.Integer, nullable=False, autoincrement=True),
+        sa.Column("subscription_id", sa.Integer, nullable=False),
+        sa.Column("org_id", sa.Integer, nullable=False),
+        sa.Column("user_id", sa.Integer, nullable=False),
+        sa.PrimaryKeyConstraint("id", name=op.f("pk_organizationrhskus")),
+        sa.ForeignKeyConstraint(
+            ["user_id"], ["user.id"], name=op.f("fk_organizationrhskus_userid")
+        ),
+        sa.ForeignKeyConstraint(["org_id"], ["user.id"], name=op.f("fk_organizationrhskus_orgid")),
+    )
+    op.create_index(
+        "organizationrhskus_subscription_id", "organizationrhskus", ["subscription_id"], unique=True
```

Comment:
Shouldn't org id also be a foreign key?

I would add some composite index too: 
- (subscription_id, org_id)
- (subscription_id, org_id, user_id)

Reply:
Is org id not a foreign key here? I thought it would just be another reference to the user table